File: Vib_Inman_8.10.jl
using LinearAlgebra
using Roots
K = [2 -1; -1 1]
M = [4 1;1 2]
a = (2*7e10)
b = (2700/12)
c(w) =  det((a*K)-((w^2)*(b*M)))
find_zeros(c,0,1000000)

File: Vib_Inman_8.10.m
clc
clear all
syms w
K = [2 -1; -1 1];
M = [4 1;1 2];
a = (2*7e10);
b = (2700/12);
c = det((a*K)-((w^2)*(b*M)));
vpasolve(c,w)


